### PR TITLE
remove registry and data on uninstall

### PR DESCRIPTION
**Pull request by moshe-kabala**
- Add remove registry
- Add purge

### CODE REVIEW

File: cmd/local/uninstall.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func NewUninstallCmd() *cobra.Command {
-
+	var purge = false
```
Comment (YuvalFeldman) [inline cmd/local/uninstall.go:11]: The purge var already gets a default false value from the flag 
```suggestion
	var purge
```

File: cmd/local/uninstall.go [hunk 2]
```diff
@@ -28,12 +28,25 @@ func NewUninstallCmd() *cobra.Command {
 				return err
 			}
 
+			err = k3d.UninstallRegister()
```
Comment (YuvalFeldman) [inline cmd/local/uninstall.go:31]: Do we always want to uninstall the register or only when we purge / deleteregister flags are true?

File: pkg/docker/utils.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+	for _, container := range containers {
+		for _, name := range container.Names {
+			if name == "/"+containerName {
+				return container.ID, nil
+			}
+		}
+	}
+
+	return "", fmt.Errorf("container '%s' not found", containerName)
```
Comment (YuvalFeldman) [inline pkg/docker/utils.go:29]:
```suggestion
	return "", log.Errorf("container '%s' not found", containerName)
```

File: pkg/k3d/registry.go [hunk 4]
```diff
@@ -103,6 +104,37 @@ func createRegistryConfig(port uint, volumes []string) *Registry {
...
+		log.Infof("Registry '%s' not found", REGISTRY_NAME)
+		log.SendCloudReport("info", "Not found registry", "Running", &map[string]interface{}{"registryName": REGISTRY_NAME})
+
+		return nil
+	}
+	log.Infof("Deleting registry %s", REGISTRY_NAME)
+
+	cli, err := docker.CreateDockerCli()
+	if err != nil {
+		return fmt.Errorf("Error creating Docker client: %v", err)
```
Comment (YuvalFeldman) [inline pkg/k3d/registry.go:120]:
```suggestion
		return log.Errorf("Error creating Docker client: %v", err)
```

**Review by YuvalFeldman — APPROVED** [review]
LGTM

File: cmd/local/uninstall.go (hunk 1 above)
Comment (moshe-kabala) [reply cmd/local/uninstall.go:11]: 'var purge bool'

File: cmd/local/uninstall.go (hunk 2 above)
Comment (moshe-kabala) [reply cmd/local/uninstall.go:31]: yes

File: pkg/k3d/registry.go (hunk 4 above)
Comment (moshe-kabala) [reply pkg/k3d/registry.go:120]: It is creating an error (not print)

**Pull request merged**